internal/x11/mappings.go

```diff
@@ -179,6 +180,7 @@ var KeycodesMc = map[string]xproto.Keycode{
 	"right":           114,
 	"up":              111,
 	"apostrophe":      48,
+	"accent.grave":	   49,
```

Comment:
This seems to be `grave.accent` on both versions I tested? (1.16.1 and 1.20.1)

Reply:
Got it